spec/services/proofing/aamva/request/verification_request_spec.rb

```diff
@@ -103,6 +103,15 @@
 
         expect(result.success?).to eq(true)
       end
+
+      it 'sends state id jurisdiction to AAMVA' do
+        applicant.state_id_data.state_id_jurisdiction = 'NY'
+        expect(
+          Nokogiri::XML(subject.body) do |config|
+            config.strict
+          end.text,
+        ).to match(/NY/)
```

Comment:
the goal of using an XML parser was likely also to navigate the XML struscture, so I'd recommend more like:

```suggestion
        doc = Nokogiri::XML(subject.body) { |config| config.strict }
        message_destination = doc.at_css('MessageDestinationId')
        expect(message_destination.text).to eq('NY')
```

Reply:
Ugh, I merged my PR before seeing your comment.  I'd modeled the change on the example you linked, which used `match`.  Oops.